app_dart/lib/src/request_handlers/get_build_status.dart

```diff
@@ -26,14 +27,15 @@ class GetBuildStatus extends RequestHandler<Body> {
   final DatastoreServiceProvider datastoreProvider;
   final BuildStatusServiceProvider buildStatusProvider;
 
-  static const String branchParam = 'branch';
+  static const String kRepoParam = 'repo';
 
   @override
   Future<Body> get() async {
     final DatastoreService datastore = datastoreProvider(config.db);
     final BuildStatusService buildStatusService = buildStatusProvider(datastore);
-    final String branch = request!.uri.queryParameters[branchParam] ?? 'master';
```

Comment:
Is this obsolete?

Reply:
Yes. Tree status is only used for tip of tree. For release branches the logic is different, and we do it by hand (the latest row is all green)